src/main/java/mate/academy/rickandmorty/service/ClientService.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.rickandmorty.service;
+
+import java.util.List;
+import java.util.Random;
+import mate.academy.rickandmorty.exception.DataProcessException;
+import mate.academy.rickandmorty.model.Character;
+import mate.academy.rickandmorty.repository.CharacterRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ClientService {
+    private static final long NUMBER_OF_CHARACTERS = 20;
+    @Autowired
+    private CharacterRepository characterRepository;
```

Comment:
use  `@RequiredArgsConstructor` for constructor-based injection, it's better way
```suggestion
    private final CharacterRepository characterRepository;
```